ant-networking/src/driver.rs

```diff
@@ -254,7 +253,7 @@ pub(super) struct NodeBehaviour {
     pub(super) blocklist:
         libp2p::allow_block_list::Behaviour<libp2p::allow_block_list::BlockedPeers>,
     pub(super) identify: libp2p::identify::Behaviour,
-    #[cfg(feature = "local")]
+    /// mDNS behaviour to use in local mode
```

Comment:
Should we not make this a `Toggle`? Then disable it completely with non-local mode to prevent home network noise.

Reply:
If you look at the implementation I check if we are in local mode before executing anything. But not sure if it does anything else? Not too familiar with how Behaviors work.

Reply:
So if a public node is running on a home network it will still broadcast its address to a local test network's nodes. Shouldn't be a problem because of different network IDs/identifiers, but would still add a lot of noisy logs for those local nodes I presume.

Reply:
Mmmh I see, that's not ideal. @dirvine mentioned getting rid of mdns all together eventually, maybe it's not worth the effort now if we plan to remove it altogether?

Reply:
Wouldn't mind getting it out, but `Toggle` is fine with me too. You can use the UPnP behavior as an example which is a bunch of lines below mDNS in the `build` method.